lambda-code/audit-logs-archiver/Dockerfile

```
@@ -0,0 +1,21 @@
+FROM public.ecr.aws/lambda/nodejs:18 as builder
+
+WORKDIR /usr/app
+
+RUN npm install -g yarn
+
+COPY package.json yarn.lock ./
+
+RUN yarn install
```

If you wanted to make your dependency installs more reproducible, you could switch things up like so:
```suggestion
RUN yarn install --frozen-lockfile
```
That way the Docker image build would fail if the install would cause an update to the lock file.